tests/Unit/RequestParserTest.php

```diff
@@ -148,7 +149,6 @@ public function nonsensicalContentTypes(): iterable
         yield ['application/foobar'];
         yield ['application/josn'];
         yield ['application/grapql'];
-        yield ['application/foo;charset=application/json'];
```

Comment:
In laravel, $request->isJson() for the following header returns true, because only /json or +json substring is checked in the Content-Type value

See https://github.com/laravel/framework/blob/9.x/src/Illuminate/Http/Concerns/InteractsWithContentTypes.php#L14-L17

Reply:
Stuff like this is why I preferred doing this ourselves...